clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/agents/TitusStreamingUpdateAgent.java

```diff
@@ -60,12 +61,10 @@
 import static com.netflix.spinnaker.clouddriver.titus.caching.Keys.Namespace.*;
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
 
+@Slf4j
```

Comment:
Why did you flip to using lombok? I prefer defining the log as it was before

Reply:
to be consistent with the other codes